Support for 1.20.

Thank you for the plugin. Please add support for 1.20
My error log...
```

[23:04:26 ERROR]: Error occurred while enabling EvenMoreFish v1.6.11.3 (Is it up to date?)
com.oheers.fish.utils.nbt.changeme.nbtapi.NbtApiException: [?]Error while calling the method 'getTagType(java.lang.String)', loaded: true, Enum: COMPOUND_GET_TYPE, Passed Class: class java.lang.Boolean
	at com.oheers.fish.utils.nbt.changeme.nbtapi.utils.nmsmappings.ReflectionMethod.run(ReflectionMethod.java:358) ~[EvenMoreFish-1.6.11.3.jar:?]
	at com.oheers.fish.utils.nbt.changeme.nbtapi.NBTReflectionUtil.getData(NBTReflectionUtil.java:635) ~[EvenMoreFish-1.6.11.3.jar:?]
	at com.oheers.fish.utils.nbt.changeme.nbtapi.NBTCompound.getType(NBTCompound.java:1008) ~[EvenMoreFish-1.6.11.3.jar:?]
	at com.oheers.fish.utils.nbt.changeme.nbtapi.NBTCompound.addCompound(NBTCompound.java:665) ~[EvenMoreFish-1.6.11.3.jar:?]
	at com.oheers.fish.FishUtils.get(FishUtils.java:229) ~[EvenMoreFish-1.6.11.3.jar:?]
	at com.oheers.fish.utils.ItemFactory.checkHead64(ItemFactory.java:162) ~[EvenMoreFish-1.6.11.3.jar:?]
	at com.oheers.fish.utils.ItemFactory.getType(ItemFactory.java:94) ~[EvenMoreFish-1.6.11.3.jar:?]
	at com.oheers.fish.utils.ItemFactory.<init>(ItemFactory.java:42) ~[EvenMoreFish-1.6.11.3.jar:?]
	at com.oheers.fish.fishing.items.Fish.<init>(Fish.java:75) ~[EvenMoreFish-1.6.11.3.jar:?]
	at com.oheers.fish.fishing.items.Names.loadRarities(Names.java:69) ~[EvenMoreFish-1.6.11.3.jar:?]
	at com.oheers.fish.EvenMoreFish.onEnable(EvenMoreFish.java:157) ~[EvenMoreFish-1.6.11.3.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189) ~[paper-1.20.1.jar:git-Paper-18]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[paper-1.20.1.jar:git-Paper-18]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:562) ~[paper-1.20.1.jar:git-Paper-18]
	at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:473) ~[paper-1.20.1.jar:git-Paper-18]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:636) ~[paper-1.20.1.jar:git-Paper-18]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:435) ~[paper-1.20.1.jar:git-Paper-18]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[paper-1.20.1.jar:git-Paper-18]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1101) ~[paper-1.20.1.jar:git-Paper-18]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:318) ~[paper-1.20.1.jar:git-Paper-18]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.IllegalArgumentException: java.lang.ClassCastException@62861afd
	at jdk.internal.reflect.GeneratedMethodAccessor39.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at com.oheers.fish.utils.nbt.changeme.nbtapi.utils.nmsmappings.ReflectionMethod.run(ReflectionMethod.java:355) ~[EvenMoreFish-1.6.11.3.jar:?]
	... 22 more
```
¡Muchas gracias! para este plugin)

--- a/pom.xml
+++ b/pom.xml
@@ -58,7 +58,7 @@
                             <shadedPattern>com.oheers.evenmorefish.bstats</shadedPattern>
                         </relocation>
                         <relocation>
-                            <pattern>de.tr7zw</pattern>
+                            <pattern>de.tr7zw.changeme.nbtapi</pattern>
                             <shadedPattern>com.oheers.fish.utils.nbt</shadedPattern>
                         </relocation>
                     </relocations>

